bucket/docker.json

```diff
@@ -22,7 +22,7 @@
         ]
     ],
     "checkver": {
-        "url": "https://github.com/docker/toolbox/releases/latest",
+        "github": "https://github.com/docker/toolbox/releases/latest",
```

Comment:
Please remove the `/releases/latest` and the `re` line, as they are unneeded. Thx.

Reply:
re is needed due to the `v` in the release tag, the /releases/latest was removed with the [commit](https://github.com/lukesampson/scoop/pull/2235/commits/3a2b6fae07f9d16887c4200bde256008b6305a3c) after.

Reply:
`re` is not needed, as checkver already has that regex. See https://github.com/lukesampson/scoop/blob/master/bin/checkver.ps1#L56

Reply:
Ahh, sorry. it's the `-ce` and possibly `-rc` parts of their version naming that the default regex doesn't catch. I could add `-ce` to the download urls?

Reply:
You’re right, my bad. Let’s get this puppy merged.